backend/.env.dist.local

```diff
@@ -155,3 +155,6 @@ CROWD_UNLEASH_DB_PORT=
 CROWD_UNLEASH_DB_USERNAME=
 CROWD_UNLEASH_DB_PASSWORD=
 CROWD_UNLEASH_DB_DATABASE=
+
+# Weekly emails settings
+CROWD_WEEKLY_EMAILS_ENABLED=true
```

Comment:
Not sure if it changes a thing practically but, since in configuration it's a string let's also set it with quotes here